### DISCUSSION TITLE
Java Vs C#继承实现多态

### DISCUSSION THREAD

**Discussion by giscus** [ichengzi/ichengzi.github.io#53] [Announcements]
# Java Vs C#继承实现多态

java vs c#， 继承实现多态

https://ichengzi.github.io/2022/07/09/java-vs-c-%E7%BB%A7%E6%89%BF%E5%AE%9E%E7%8E%B0%E5%A4%9A%E6%80%81.html

**Comment by ichengzi** [comment]
倒没有更喜欢哪种方式， 但是 java 默认 `override` 增大了理解继承链路的难度， 需要借助 IDE 才能方便理解继承链路深的类
